docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Building for TV
+description: A guide for building an Expo app for an Apple TV or Android TV target.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+> **Warning** TV development is available only in SDK 50 and later, and is still considered experimental. Not all Expo features and SDK modules are available on TV.
+
+React Native is supported on Apple TV and Android TV through the [React Native TV project](https://github.com/react-native-tvos/react-native-tvos).  It is not just for TV — it is a full fork of the core repo, and supports both phone and TV targets, including Hermes and Fabric support
+
+By taking an Expo project and using the React Native TV package as the `react-native` dependency, the project can be made to target either mobile (Android, iOS) or TV (Android TV, Apple TV) devices.
+
+The changes needed to the native Android and iOS files are minimal, and can be automated with a [config plugin](https://github.com/react-native-tvos/config-tv/tree/main/packages/config-tv) if you use [prebuild](/workflow/prebuild/). The following is a list of changes that the config plugin makes, which you can alternatively apply manually:
+
+- iOS:
+  - **ios/Podfile** is modified to target tvOS instead of iOS
+  - The Xcode project is modified to target tvOS instead of iOS
+  - The splash screen (**SplashScreen.storyboard**) is modified to work on tvOS
+- Android:
+  - **AndroidManifest.xml** is modified
+  - The default phone portrait orientation is removed
+  - The required intent for TV apps is added
+  - **MainApplication.kt** is modified to remove unsupported Flipper invocations
+
+## Quick start
+
+The fastest way to generate a new project is described in the [TV Example](https://github.com/expo/examples/tree/master/with-tv) in the Expo examples repo.
+
+## Step by step
+
+If you already have an existing Expo project, this walkthrough shows what is required to take an Expo project and modify it for TV.
+
+You will need:
+
+- An Expo project using SDK 50 or later.
+- **Apple TV**:
+  - Xcode 15 or later
+  - tvOS SDK 17 or later. (This is not installed automatically with Xcode. If you did not install it at the prompt when first opening Xcode, you can install later with `xcodebuild -downloadAllPlatforms`).
+- **Android TV**:
+  - In the Android Studio SDK manager, select the dropdown for the Android SDK you are using (API version 31 or later), and make sure an Android TV system image is selected for installation. (For Apple Silicon Macs, choose the ARM 64 image, otherwise, choose the Intel x86_64 image).
+  - After installing the Android TV system image, create an Android TV emulator using that image (the process is the same as for creating an Android phone emulator).
```

Comment:
```suggestion
- **Android TV**:
  - In the Android Studio SDK manager, select the dropdown for the Android SDK you are using (API version 31 or later), and make sure an Android TV system image is selected for installation. (For Apple silicon, choose the ARM 64 image. Otherwise, choose the Intel x86_64 image).
  - After installing the Android TV system image, create an Android TV emulator using that image (the process is the same as creating an Android phone emulator).
- **Apple TV**:
  - Xcode 15 or later.
  - tvOS SDK 17 or later. (This is not installed automatically with Xcode. You can install it later with `xcodebuild -downloadAllPlatforms`).
```

- Apple platform order formatting
-  Let's make the last point in Apple TV section to be concise to provide instructions to install the tvOS since we are already mentioning that apple it is not automatically installed
- Let's use lowercase for "silicon" when referencing "Apple silicon". Apple docs follow this convention and I think we should too (for example: https://developer.apple.com/documentation/apple-silicon &mdash; ignore the title of the page since their docs follow Title casing for section/page headers)